frontend/packages/dev-console/src/components/import/pipeline/PipelineTemplate.tsx

```diff
@@ -8,16 +8,27 @@ import { CLUSTER_PIPELINE_NS } from '../../../const';
 import { PipelineModel } from '../../../models';
 import PipelineVisualization from '../../pipelines/detail-page-tabs/pipeline-details/PipelineVisualization';
 import { Pipeline } from '../../../utils/pipeline-augment';
+import { NormalizedBuilderImages } from '../../../utils/imagestream-utils';
+import { ReadableResourcesNames } from '../import-types';
 
 const MISSING_DOCKERFILE_LABEL_TEXT =
   'The pipeline template for Dockerfiles is not available at this time.';
-const MISSING_RUNTIME_LABEL_TEXT = 'There are no pipeline templates available for this runtime.';
 
 const labelType = 'pipeline.openshift.io/type';
 const labelRuntime = 'pipeline.openshift.io/runtime';
 const labelDocker = 'pipeline.openshift.io/strategy';
 
-const PipelineTemplate: React.FC = () => {
+const getAlertText = (isDockerStrategy, builderImage, resourceType) => {
```

Comment:
Nit, let's type it for sanity sake.
```suggestion
const getAlertText = (isDockerStrategy: boolean, builderImage: string, resourceType: string): string => {
```

Reply:
Done